tests/framework/artifacts.py

```diff
@@ -366,19 +367,33 @@ def microvms(self, keyword=None):
             keyword=keyword
         )
 
-    def firecrackers(self, keyword=None):
+    def firecrackers(self, keyword=None, older_than=None):
         """Return fc/jailer artifacts for the current arch."""
-        return self._fetch_artifacts(
+        firecrackers = self._fetch_artifacts(
             ArtifactCollection.ARTIFACTS_BINARIES,
             ArtifactCollection.FC_EXTENSION,
             ArtifactType.FC,
             FirecrackerArtifact,
             keyword=keyword
         )
 
-    def firecracker_versions(self):
+        if older_than is not None:
```

Comment:
A clarification comment would be useful.

Reply:
done